packages/vx-demo/src/docs-v2/examples/vx-axis/Example.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import AreaClosed from '@vx/shape/lib/shapes/AreaClosed';
+import { Grid } from '@vx/grid';
+import { curveMonotoneX } from '@vx/curve';
+import { scaleUtc, scaleLinear, scaleLog, scaleBand } from '@vx/scale';
+import { AxisBottom } from '@vx/axis';
+import { LinearGradient } from '@vx/gradient';
+import { timeFormat } from 'd3-time-format';
+
+import './styles.css';
```

Comment:
I think this probably should actually be moved to index since it's _only_ for codesandbox